datalad_container/extractors/metalad_container.py

```diff
@@ -0,0 +1,85 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
```

Comment:
@yarikoptic is there a reason we set noexpand tab? Seems wrong for python.

Reply:
yeah, I think this some historical artifact. Feel welcome to fix everywhere (in separate PR) in all extensions . I think we did similar in datalad core repo but even there not everywhere

```shell
❯ git grep -h '^# ex:' | sort | uniq -c
    264 # ex: set sts=4 ts=4 sw=4 et:
     14 # ex: set sts=4 ts=4 sw=4 noet:
```

so here is a little good menial contribution to make ;)